src/Tizen.Network.Bluetooth/Tizen.Network.Bluetooth/BluetoothAvrcpControl.cs

```diff
@@ -0,0 +1,611 @@
+        public Track GetTrackInfo()
+        {
+            if (BluetoothAdapter.IsBluetoothEnabled && Globals.IsInitialize)
+            {
+                return BluetoothAvrcpControlImpl.Instance.GetTrackInfo();
+            }
+            else
+            {
+                BluetoothErrorFactory.ThrowBluetoothException((int)BluetoothError.NotEnabled);
+            }
+            return null;
+        }
+        public void FreeTrackInfo(Track trackData) //needs special testing
+        {
+
+        }
+
+        /// <summary>
+        /// Sends a particular play command to the target device
+        /// </summary>
+        /// <remarks>
+        /// The remote device must be connected.
+        /// </remarks>
+        /// <param name="command">Command to be sent</param>
+        /// <exception cref="NotSupportedException">Thrown when the Bluetooth is not supported.</exception>
+        /// <exception cref="InvalidOperationException">Thrown when the Bluetooth is not enabled
+        /// or when sending command to the target device fails.
+        /// </exception>
+        /// <since_tizen> 8 </since_tizen>
+        public void SendPlayerCommand(PlayerCommand command)
+        {
+            if (BluetoothAdapter.IsBluetoothEnabled && Globals.IsInitialize)
+            {
+                BluetoothAvrcpControlImpl.Instance.SendPlayerCommand(command);
+            }
+            else
+            {
+                BluetoothErrorFactory.ThrowBluetoothException((int)BluetoothError.NotEnabled);
+            }
+        }
+
+        /// <summary>
+        /// Sends a play command to a particular target device
+        /// </summary>
+        /// <remarks>
+        /// The remote device must be connected.
+        /// </remarks>
+        /// <param name="command">Command to be sent</param>
+        /// <param name="remote_address">Address of the device to send command</param>
+        /// <exception cref="NotSupportedException">Thrown when the Bluetooth is not supported.</exception>
+        /// <exception cref="InvalidOperationException">Thrown when the Bluetooth is not enabled
+        /// or when sending command to the target device fails.
+        /// </exception>
+        /// <since_tizen> 8 </since_tizen>
+        public void SendPlayerCommandTo(PlayerCommand command, string remote_address)
+        {
+            if (BluetoothAdapter.IsBluetoothEnabled && Globals.IsInitialize)
+            {
+                BluetoothAvrcpControlImpl.Instance.SendPlayerCommandTo(command, remote_address);
+            }
+            else
+            {
+                BluetoothErrorFactory.ThrowBluetoothException((int)BluetoothError.NotEnabled);
+            }
+        }
+
+        /// <summary>
+        /// Sets absolute volume of target device
+        /// </summary>
+        /// <remarks>
+        /// The remote device must be connected
+        /// </remarks>
+        /// <param name="volume">The volume level to be set</param>
```

Comment:
The range is defined internally. (about 0~63, not correct)
Because this function is for internal, first of all, don't write the range and will be updated in later.

Reply:
https://www.google.co.kr/url?sa=t&source=web&rct=j&url=https://www.bluetooth.org/docman/handlers/DownloadDoc.ashx%3Fdoc_id%3D119996&ved=2ahUKEwiFvOqJsojqAhVFL6YKHbjACnYQFjAAegQIARAB&usg=AOvVaw19jecb_Ei26Q3lsr5DuWuq

from a spec documemt, absolute volume level is represent as percent 0-100

Reply:
Thank you. The value can be 0x00 ~ 0x7F. I will update.

Reply:
Why not 0~100??

Reply:
The volume is specified as a percentage of the maximum. The value 0x0 corresponds to 0%. The value 0x7F corresponds to 100%.

volume level will be checked like below in code level.
    if (volume > 127) {
        g_dbus_pending_property_error(id,
                    ERROR_INTERFACE ".InvalidArguments",
                    "Invalid arguments in method call");
        return;
    }